### PR TITLE
[Merged by Bors] - chore(LinearAlgebra/DFinsupp): remove dependency on `Basis`

### PR DESCRIPTION
**Pull request by eric-wieser**
The motivation here is to explore re-defining `Basis` with `DFinsupp` instead of `Finsupp`, in order to make it computable.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/LinearAlgebra/FinsuppVectorSpace.lean [hunk 1]
```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johannes Hölzl
 -/
 import Mathlib.LinearAlgebra.StdBasis
+import Mathlib.LinearAlgebra.DFinsupp
```
Comment (Vierkantor) [inline Mathlib/LinearAlgebra/FinsuppVectorSpace.lean:7]:
```suggestion
import Mathlib.LinearAlgebra.DFinsupp
import Mathlib.LinearAlgebra.StdBasis
```

**Review by Vierkantor — APPROVED** [review]
Sounds like a good plan, but please don't forget to fix the build errors!

**Comment by eric-wieser** [comment]
@Vierkantor: CI is happy

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**